layers/+chat/circe/packages.el

```diff
@@ -0,0 +1,142 @@
+;;; packages.el --- Circe layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
+;;
+;; Author: Chris Barrett <chris.d.barrett@me.com>
+;; Author: Aidan Nyquist <contact@aidannyquist.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Code:
+
+(defconst circe-packages
+  '(circe
+    circe-notifications
+    persp-mode
+    helm
+    helm-circe
+    ))
+
+(defun circe/init-circe ()
+  (use-package circe
+    :commands circe
+    :config
+
+    (set-face-background 'circe-prompt-face nil)
+    (setq circe-reduce-lurker-spam t)
+    (setq circe-use-cycle-completion t)
+    (setq circe-active-users-timeout (* 60 30)) ; 30 minutes
+    (setq circe-format-say "{nick}> {body}")
+    (setq circe-format-self-say (concat (propertize ">>>" 'face 'circe-self-say-face) " {body}"))
+    (setq circe-prompt-string (concat (propertize ">>>" 'face 'circe-prompt-face) " "))
+    (setq circe-highlight-nick-type 'all)
```

Comment:
Can the `setq` forms be moved to `:init`? That way, users can set them in `dotspacemacs/user-config`.  By setting them in `:config`, you overwrite any user-specified settings.